### PR TITLE
DBZ-3658 Oracle JDBC driver JDK8 to JDK11 (ojdbc11) 21.15.0.0

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-3658

@obabec your favorite task -- we'll need to align the testing farm to contain `ojdbc11` 21.11.0.0, when you have time :grinning: 

- [x] Check release bits and verify no changes needed for `ojdbc8` to `ojdbc11`.
- [x] Update driver reference used by the JDBC sink connector.

### CODE REVIEW

**Comment by Naros** [comment]
@roldanbob wrt docs, I wonder if we shouldn't just change the attribute from `ojdbc8-version`, now `ojdbc11-version` to actually just reading `ojdbc-version`.  wdyt?  Would this be easy to coordinate for downstream?

**Comment by roldanbob** [comment]
> @roldanbob wrt docs, I wonder if we shouldn't just change the attribute from `ojdbc8-version`, now `ojdbc11-version` to actually just reading `ojdbc-version`. wdyt? Would this be easy to coordinate for downstream?

We can easily update the attribute name from `ojdbc-version` to `ojdbc-version`, I think , and then assign the current value to it. That makes sense to me. Upstream, it looks like there are only 3 instances in the sources where we reference the attribute, so it's a pretty painless change.

**Comment by Naros** [comment]
Sounds good, then I'll modify the PR based on that.  Thanks @roldanbob.

**Comment by roldanbob** [comment]
@Naros Based on the Jira issue, I assume that this affects only 2.6 and later, right? Nothing to turn around for 2.5?

**Comment by Naros** [comment]
> @Naros Based on the Jira issue, I assume that this affects only 2.6 and later, right? Nothing to turn around for 2.5?

That's correct, and it may even be something we hold off until 2.7; deferring that to @jpechane.
Given that we're on the brink of 2.6.CR1, I'd say that may even be the best and safest bet.

**Comment by jpechane** [comment]
@Naros Where do we stand with this PR? Can it be undrafted for 3.0?

**Comment by Naros** [comment]
Let me rerun the checks and get back to you on that @jpechane

**Comment by roldanbob** [comment]
> @roldanbob wrt docs, I wonder if we shouldn't just change the attribute from `ojdbc8-version`, now `ojdbc11-version` to actually just reading `ojdbc-version`. wdyt? Would this be easy to coordinate for downstream?

@Naros Per our earlier convo, should we use the attribute `ojdbc-version` rather than `ojdbc11-version`?

**Comment by Naros** [comment]
I've tested both XStream and LogMiner locally.  I'll need to coordinate with Ondrej so that we can update the testing farm images before we merge this PR so that we don't run into an inconsistency of dependencies, but otherwise the tests passed for both adapters using ojdbc11 21.15.0.0.

**Review by vjuranek — APPROVED** [review]

**Comment by Naros** [comment]
Thanks for all the help on making this happen for Debezium 3 @vjuranek ❤️

**Pull request merged**
